.env.example

```
@@ -1,2 +1,12 @@
 SECURITY_ENC_KEY_BASE64=
-SECURITY_HASH_PEPPER=
\ No newline at end of file
+SECURITY_HASH_PEPPER=
+LOCAL_MAIL_HOST=email-smtp.ap-northeast-2.amazonaws.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LOCAL_MAIL_HOST key should go before the SECURITY_ENC_KEY_BASE64 key

(UnorderedKey)